stdlib/public/stubs/SwiftNativeNSXXXBase.mm.gyb

```diff
@@ -87,22 +87,6 @@ swift_stdlib_NSObject_isEqual(id lhs,
   return (lhs == rhs) || [lhs isEqual:rhs];
 }
 
-SWIFT_CC(swift) SWIFT_RUNTIME_STDLIB_SPI
```

Comment:
Oh, getting rid of that is really nice :-) That halves the number of class lookup-by-string calls we're doing.